### DISCUSSION TITLE
Your emoji use is not how you perceive it

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9941] [General Discussions]
Hello,

Please read: 

[https://gitlab.com/gitlab-org/gitlab/-/issues/353392](https://gitlab.com/gitlab-org/gitlab/-/issues/353392)

For many Godot users posting here is their first time using Github. Thumbs down and confused are useful, fast, and have meaning for those of us who use Github regularly in a professional environment, but for new users it is driving them away and toxic to the "fun, friendly" atmosphere Godot claims to aim for. Often perceived as elitist/bullying/no one wants to deal with that experience, and many offices, mine included, ban their use.

Please consider not using these negative emojies if you have nothing constructive to add or we should make an FAQ to explain their use to new users so it is not misunderstood and taken the wrong way.

Something is driving away new users who try to help and this isn't it but it isn't helping either.

**Comment by jmarceno** [comment]
Strongly disagree. 

I understand that we use them as "votes" in this repo and if you can vote in favor, someone should be able to vote against it.

As with any opposition, people should learn that it is not a personal attack, just a disagreement on the subject.

Not every "vote" should need an explanation. Neither positive or negative.
